draft-vvv-webtransport-http3.md

```diff
@@ -157,10 +157,61 @@ is closed.
 
 ## Establishing a Transport-Capable HTTP/3 Connection
 
+In WebTransport, an *exclusive mode* is a state of HTTP/3 connection where the
+connection is expected to have exactly one WebTransport session, and no other
+HTTP requests.
+
 In order to indicate support for WebTransport, both the client and the server
-MUST send a SETTINGS_ENABLE_WEBTRANSPORT value set to "1" in their SETTINGS
-frame.  Endpoints MUST NOT use any WebTransport-related functionality unless
-the parameter has been negotiated.
+MUST send a SETTINGS_ENABLE_WEBTRANSPORT setting.  The value of
+SETTINGS_ENABLE_WEBTRANSPORT is "0" (Unsupported), however it can also assume
+values of "1" (Supported) and "2" (Exclusive).  The exact meaning of the value
+depends on the party.
+
+SETTINGS_ENABLE_WEBTRANSPORT sent by the client has the following possible values:
+* Unsupported ("0") indicates that the client does not support WebTransport
+  over HTTP/3.
+* Supported ("1") indicates that the client supports WebTrasnport over HTTP/3
+  and may potentially initiate a WebTransport session in the future.
+* Exclusive ("2") indicates that the client both supports WebTransport over
+  HTTP/3 and has a WebTransport request that it intends to send over the
+  connection in question.
+
+SETTINGS_ENABLE_WEBTRANSPORT sent by the server has the following possible values:
+* Unsupported ("0") indicates that the server does not support WebTransport
+  over HTTP/3.
+* Supported ("1") indicates that the server supports WebTrasnport over HTTP/3.
+* Exclusive ("2") indicates that the server expects to handle exactly one
+  request over the connection in question, and that request has to be a
+  WebTransport session.
+
+Once an HTTP/3 implementation has the SETTINGS_ENABLE_WEBTRANSPORT from both
+sides, the following should happen:
+
+* If one of client's or server's value is "0", and another value is either "0"
```

Comment:
This doesn't handle the case where client sends 0 and server sends 2. Why not simply say: if any endpoint sent 0 then WebTransport is not supported on this connection - which means clients MUST NOT send the special CONNECT request, and servers MUST fail it with a certain error.

Reply:
No, it handles it with a connection error (third bullet point).